### PR TITLE
Override graphene.JSONString to have more meaningful message in error message

### PR DESCRIPTION
**Pull request by L3str4nge**
Override `graphene.JSONString.parse_literal` function to catch error when input is the invalid json. Unfortunately there is no information about field name because it's a `@staticmethod` but it will be more readable for users either.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

File: saleor/graphql/product/fields.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from json import JSONDecodeError
+
+import graphene
+from graphql.error import GraphQLError
+
+
+class JSONStringWithValidation(graphene.JSONString):
```
Comment (fowczarek) [inline saleor/graphql/product/fields.py:7]: Should we use this field for `Page.content` field?

Comment (L3str4nge) [reply saleor/graphql/product/fields.py:7]: If we want to have more readable error for invalid JSONString fields I guess we have to change all occurrences.

Comment (L3str4nge) [reply saleor/graphql/product/fields.py:7]: Replacing is possible, gonna do it soon. For keep namings there is a problem with generating schema about different types with the same names but I will try to replace it somehow :)

Comment (patrys) [reply saleor/graphql/product/fields.py:7]: I think we already use a custom scalar for built-in floats?

**Review by maarcingebala — APPROVED** [review]

**Review by IKarbowiak — APPROVED** [review]

**Comment by maarcingebala** [comment]
@L3str4nge Please fix conflicts and add changelog :)

**Review by korycins — APPROVED** [review]

**Pull request merged**